Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM openjdk
+
+# Install libsodium
+RUN apt-get update && apt-get install -y \
+        build-essential \
+        software-properties-common
```

Comment:
Is `software-properties-common` required? It compiles without it and it drags in a ton of weird stuff including exim (a mailserver!)

Reply:
I couldn't run `add-apt-repository` without it, but if we're not using that then can be removed again